arangod/VocBase/LogicalView.cpp

```diff
@@ -405,27 +405,7 @@ Result LogicalView::rename(std::string&& newName) {
     return Result();  // NOOP
```

Comment:
Is this return still needed?

Reply:
Good catch. I think it can be safely removed as well. Will adjust the PR.